app/src/main/java/vn/sunasterisk/realfootball/FootballApplication.kt

```diff
@@ -0,0 +1,19 @@
+package vn.sunasterisk.realfootball
+
+import android.app.Application
+import android.content.Context
+
+class FootballApplication : Application() {
+
+    override fun onCreate() {
+        super.onCreate()
+        INSTANCE = this
+    }
+
+    companion object {
+        lateinit var INSTANCE: FootballApplication
```

Comment:
INSTANCE để public sao?

Reply:
Cái ý cx tuỳ vào mục đích mìh dùng thôi
Để public đơn giản là muốn dùng trực tiếp nó ko cần thông qua get set thôi

Reply:
theo ông thì mình có nên để private set cho thuộc tính này không để tránh thay đổi từ bên ngoài hay là ông muốn biến này có thể thay đổi từ bên ngoài được nhỉ? 
```suggestion
        lateinit var INSTANCE: FootballApplication
            private set
```

Reply:
t đang để private set mà nhỉ . chưa hiểu ý ô lắm